src/Makefile

```diff
@@ -130,52 +140,60 @@ depend: $(addsuffix /depend, $(SUBDIRS))
 ext_depend: check_portaudio
 	-for x in $(EXT_SUBDIRS); do $(MAKE) -C $$x depend; done
 
+# 'make libs' to skip binaries, and build only libraries.
+.PHONY: libs $(SUBDIRS_LIBS)
```

Comment:
SUBDIRS_LIBS only appears these two times, elsewhere it's SUBDIRS_LIB; you didn't mean SUBDIRS_LIB?  same with SUBDIRS_BINS?

Reply:
Done. Thanks fro spotting.